huaweicloud/services/cbr/resource_huaweicloud_cbr_policy.go

```diff
@@ -144,6 +151,13 @@ func ResourcePolicy() *schema.Resource {
 							Optional:    true,
 							Description: "The latest backup of each year is saved in the long term.",
 						},
+						"full_backup_interval": {
```

Comment:
this parameter not contained in buildPolicyOpDefinition

Reply:
fixed, the problem has been occured in the latest rebase option